CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [1.0.2] - 2023-11-29
 
 ### Added
-- Added an model card in the root
+- Added a model card in the root
```

Comment:
Strictly speaking no longer in the root and maybe add a ### Changed category with moved the dataset card

Reply:
I was too quick